src/main/resources/commands.json

```diff
@@ -159,11 +164,29 @@
   {
     "command": "op-me",
     "description": "want some OP?",
-    "localChat": "<gray>[<dark_red><bold>!</bold><gray>] Made <name> a Server Operator."
+    "localChat": "<gray>[<dark_red><bold>!</bold><gray>] Made <name> a Server Operator.",
+    "othersChat": "<gold><name> <yellow>tried to hack OP!"
+  },
+  {
+    "command": "bam",
+    "description": "somebody add bam!",
+    "localChat": "<yellow><name> left the Network.",
```

Comment:
I would prefer for FunCommands messages to not get confused with _actual_ proxy messages. I do have an idea to rectify this concern, but that will have to wait until after this PR.